Components/Button/button.css

```diff
@@ -0,0 +1,91 @@
+.btn {
+  display: inline-block;
+  font-weight: 400;
+  line-height: 1.5;
+  color: rgb(var(--color-light));
+  text-align: center;
+  text-decoration: none;
+  vertical-align: middle;
+  cursor: pointer;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  user-select: none;
+  background-color: transparent;
+  border: 1px solid transparent;
+  padding: 0.375rem 0.75rem;
+  font-size: 1rem;
+  border-radius: 0.25rem;
+  transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out,
+    border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+}
+
+.btn-primary {
+  background-color: rgb(var(--primary));
+  border-color: rgb(var(--primary));
+}
+
+.btn-secondary {
+  background-color: rgb(var(--secondary));
+  border-color: rgb(var(--secondary));
+}
+
+.btn-success {
+  background-color: rgb(var(--success));
+  border-color: rgb(var(--success));
+}
+
+.btn-danger {
+  background-color: rgb(var(--danger));
+  border-color: rgb(var(--danger));
+}
+
+.btn-warning {
+  background-color: rgb(var(--warning));
+  border-color: rgb(var(--warning));
+  color: rgb(var(--color-dark));
+}
+
+.btn-info {
+  background-color: rgb(var(--info));
+  border-color: rgb(var(--info));
+  color: rgb(var(--color-dark));
+}
+
+.btn-link {
+  font-weight: 400;
+  color: rgb(var(--primary));
+  text-decoration: underline;
+}
+
+.btn.btn-icon-container {
+  padding-left: 0.8em;
+
+  /* flex - properties */
+  display: flex;
+  align-items: center;
+  gap: 0.5em;
+}
+
+.btn-svg-icons {
+  font-size: var(--btn-icon-size) !important;
+}
+
+.btn-primary .btn-svg-icons {
+  color: var(--color-dark);
+}
+
+.btn-floating {
+  width: 3rem;
+  height: 3rem;
+  border-radius: 50%;
+  border-width: 1px; /* thicker border distorts circle shape */
```

Comment:
try to give fixed position for floating button for example compose button in gmail app